src/modules/aws_ec2@0.0.1/index.ts

```diff
@@ -36,7 +36,7 @@ export const AwsEc2Module: Module = new Module({
   mappers: {
     instance: new Mapper<Instance>({
       entity: Instance,
-      entityId: (i: Instance) => i.instanceId ?? i.name,
+      entityId: (i: Instance) => i.instanceId ?? i.id?.toString() ?? '',
```

Comment:
when it could not be an i.id?

Reply:
Only when it comes from AWS, but then `instanceId` is guaranteed to exist, but I can't convince Typescript of that.